Simplified/LibraryFinder/UI/NYPLLibraryFinderLibraryCell.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  NYPLLibraryFinderLibraryCell.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2021-04-22.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import UIKit
+
+enum NYPLLibraryFinderLibraryCellType {
+  case myLibrary
+  case newLibrary
+}
+
+class NYPLLibraryFinderLibraryCell: UICollectionViewCell {
+  static let reuseIdentifier = "NYPLLibraryFinderLibraryCellReuseIdentifier"
```

Comment:
Minor suggestion here. Some time ago I created this protocol and I think we can use this.

```
public protocol Stringable {
    static var asString: String { get }
}

public extension Stringable {
    var asString: String {
        return String(describing: type(of: self))
    }

    static var asString: String {
        return String(describing: self)
    }
    
    static var nibName: String {
        return self.asString
    }
    
    static var reuseId: String {
        return self.asString
    }
    
    var nibName: String {
        return self.asString
    }
    
    var reuseId: String {
        return self.asString
    }
}

extension UITableViewCell: Stringable {}
extension UICollectionViewCell: Stringable {}
```

With this, everywhere we need a reuse identifier for NYPLLibraryFinderLibraryCell, we can just use `NYPLLibraryFinderLibraryCell .reuseId`.

Reply:
nice... is this for when each cell class is associated to only one collection view section?

Reply:
it can be used anywhere we need to register a collectionview/tableview cell. I've used it for view controllers as well in the past. No need to manually add identifiers .